src/cmake/modules/FindOpenVDB.cmake

```diff
@@ -89,6 +89,11 @@ if (OpenVDB_FOUND)
         set_property(TARGET OpenVDB::openvdb APPEND PROPERTY
             IMPORTED_LOCATION "${OPENVDB_LIBRARIES}")
     endif ()
+
+    # Note: OpenVDB older than 12 needs Boost headers.
+    if (OpenVDB_VERSION VERSION_LESS 12)
+        checked_find_package (Boost)
```

Comment:
I think that if this succeeds, you can just append Boost_INCLUDE_DIRS to OPENVDB_INCLUDES, and then the other change in openvdb.imageio/CMakeLists is not needed at all?

And should we bother to handle the case of if this find fails? The robust solution would be for a failure to find boost to actually set OpenVDB_FOUND to false again, maybe?